fighthealthinsurance/fax_views.py

```diff
@@ -43,14 +43,14 @@ class StageFaxView(generic.FormView):
     def form_valid(self, form):
         form_data = form.cleaned_data
         # Get all of the articles the user wants to send
-        logger.debug(f"Items {list(self.request.POST.items())}")
+        logger.debug(f"Items received: {len(list(self.request.POST.items()))}")
         pubmed_checkboxes = [
             key[len("pubmed_") :]
             for key, value in self.request.POST.items()
             if key.startswith("pubmed_") and value == "on"
         ]
         form_data["pubmed_ids_parsed"] = pubmed_checkboxes
-        logger.debug(f"Staging fax with {form_data}")
+        logger.debug(f"Staging fax with pubmed IDs count: {len(pubmed_checkboxes)}")
```

Comment:
Why lend of pubmed checkboxes?

Reply:
Fair. Logging the list of the pubmed ids is likely to be more useful while still not exposing any sensitive information since it's not tied to any user in particular. Also up on L46 we should not log all of the post items then.